docs_src/examples/Ch-ExamplesAddingMQTTDevice.md

```diff
@@ -164,73 +247,151 @@ Using the detailed script below as a simulator, there are three behaviors:
         ```   
         {"cmd":"randnum", "method":"get", "uuid":"293d7a00-66e1-4374-ace0-07520103c95f", "randnum":42.0}
         ```
-3.  Receive the set request, then change the device value.
 
+    **Using Multi-level Topic:**
+    
+    1. The simulator receives the request from the MQTT broker, the topic is `command/my-custom-device/randnum` and the message is similar to the following:
```

Comment:
the topic should add method/uuid according to the source code change.
In this document, we should state the rule of the topic definition clearly, such as {topic}/{cmd}/{method}/{cmdUuid}
the topic is not defined arbitrarily.